### PR TITLE
Implement Warped Fungus/Carrot on a Stick

### PR DESCRIPTION
**Pull request by pandaaaBE**
## Introduction
Implement Warped Fungus/Carrot on a Stick. (currently this is cosmetic only because there are no mobs in pocketmine, but I need this for my mobs plugin :P)

### API changes
`VanillaItems::CARROT_ON_A_STICK`
`VanillaItems::WARPED_FUNGUS_ON_A_STICK()`

## Tests

I tested this PR by doing the following (tick all that apply):
- [x] Playtesting using a Minecraft client (provide screenshots or a video)

### CODE REVIEW

**Comment by pandaaaBE** [comment]
grafik

File: src/item/CarrotOnAStick.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+namespace pocketmine\item;
+
+class CarrotOnAStick extends Durable{
+
+	public function getMaxStackSize() : int{
+		return 1;
+	}
+
+	public function getMaxDurability() : int{
+		return 26;
```
Comment (ShockedPlot7560) [inline src/item/CarrotOnAStick.php:33]: I find 25, is this normal ?

Comment (pandaaaBE) [reply src/item/CarrotOnAStick.php:33]: https://minecraft.wiki/w/Carrot_on_a_Stick

In Java Edition this is 25 and in Bedrock it is 26.

Comment (ShockedPlot7560) [reply src/item/CarrotOnAStick.php:33]: Thats not what this one says, need testing to be fixed ? https://minecraft.fandom.com/wiki/Carrot_on_a_Stick

Comment (pandaaaBE) [reply src/item/CarrotOnAStick.php:33]: https://github.com/PowerNukkitX/PowerNukkitX/blob/c8164b1fd104151a6e88af9133e6f5209795e1bb/src/main/java/cn/nukkit/item/ItemTool.java#L50

PowerNukkitX is using 26 as the value, but if you want I can look with a resource pack at the value later

Comment (ShockedPlot7560) [reply src/item/CarrotOnAStick.php:33]: Should be checked yes unless someone has another source?

Comment (pandaaaBE) [reply src/item/CarrotOnAStick.php:33]: image

It is 26

**Review by ShockedPlot7560 — DISMISSED** [review]

**Comment by pandaaaBE** [comment]
I think this PR is finished

**Comment by pandaaaBE** [comment]
Waiting for more pm member to approve this little pr

File: src/item/VanillaItems.php [hunk 2]
```diff
@@ -459,7 +461,11 @@ protected static function setup() : void{
 		self::register("feather", new Item(new IID(Ids::FEATHER), "Feather"));
 		self::register("fermented_spider_eye", new Item(new IID(Ids::FERMENTED_SPIDER_EYE), "Fermented Spider Eye"));
 		self::register("fire_charge", new FireCharge(new IID(Ids::FIRE_CHARGE), "Fire Charge"));
+
```
Comment (dries-c) [inline src/item/VanillaItems.php]: This next line isn't necessary, same goes for the next line underneath

**Review by dries-c — CHANGES_REQUESTED** [review]

**Review by dries-c — APPROVED** [review]

**Review by ShockedPlot7560 — APPROVED** [review]

**Pull request closed without merging**